Build failure (GCC 13)

On Fedora 38 (dev version) using GCC 13.0

```
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:43:5: error: 'uint8_t' does not name a type
   43 |     uint8_t state;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:20:1: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   19 | #include <string>
  +++ |+#include <cstdint>
   20 | 
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:44:5: error: 'uint8_t' does not name a type
   44 |     uint8_t dont_handshake;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:44:5: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:59:5: error: 'uint8_t' does not name a type
   59 |     uint8_t state;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:59:5: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:60:5: error: 'uint8_t' does not name a type
   60 |     uint8_t version;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:60:5: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:61:5: error: 'uint8_t' does not name a type
   61 |     uint8_t method;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:61:5: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:62:5: error: 'uint8_t' does not name a type
   62 |     uint8_t dns_tunnel;
      |     ^~~~~~~
/builddir/build/BUILD/php-pecl-swoole5-5.0.2/NTS/include/swoole_proxy.h:62:5: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?

```

--- a/include/swoole_proxy.h
+++ b/include/swoole_proxy.h
@@ -17,6 +17,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
 
 #define SW_SOCKS5_VERSION_CODE 0x05
 

